include/swift/AST/DiagnosticsSema.def

```diff
@@ -1740,6 +1740,8 @@ NOTE(where_requirement_failure_both_subst,none,
      "where %0 = %1, %2 = %3", (Type, Type, Type, Type))
 NOTE(requirement_implied_by_conditional_conformance,none,
      "requirement from conditional conformance of %0 to %1", (Type, Type))
+NOTE(wrapped_type_satisfies_requirement,none,
+     "wrapped type %0 satisfies this requirement", (Type))
```

Comment:
I agree that it would be better to say something actionable, but I can't think of anything better than a vague `; did you mean to unwrap?` or similar...any ideas??

Reply:
Sure, `; did you mean to unwrap?` sounds good to me!